src/app/models/Todo.ts

```
@@ -2,4 +2,5 @@ export class Todo{
     id:number;
     title:String;
     completed:boolean;
+    assignee:String;
```

Biome (1.9.4)

[error] Don't use 'String' as a type.

Use lowercase primitives for consistency.
Safe fix: Use 'string' instead


(lint/complexity/noBannedTypes)